CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 
 
+## 4.9.3 (2023-07-27)
+
+- `ERC2771Context`: Return the forwarder address whenever the `msg.data` of a call originating from a trusted forwarder is not long enough to contain the request signer address (i.e. `msg.data.length` is less than 20 bytes), as specified by ERC-2771. ([#4482](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4482))
+- `ERC2771Context`: Prevent revert in `_msgData()` when a call originating from a trusted forwarder is not long enough to contain the request signer address (i.e. `msg.data.length` is less than 20 bytes). Return the full calldata in that case.
```

Comment:
Do we not have a PR link for that one?

Reply:
The PR links break with cherry-picked commits apparently, we need to add the PR number manually. Will do now.

Reply:
https://github.com/OpenZeppelin/openzeppelin-contracts/commit/70dea74ac7d969321d56cebdb96cf31cd9f1fb8a